conmon/conmon.c

```diff
@@ -96,6 +96,8 @@ static inline void strv_cleanup(char ***strv)
 #define CMD_SIZE 1024
 #define MAX_EVENTS 10
 
+#define DEFAULT_SOCKET_PATH "/var/lib/crio"
```

Comment:
@rhatdan is this intended or it should be `/var/run/crio` ? Haven't found any references to this path

Reply:
This needs to survive a reboot I believe.  As I recall. I think this is used cleanup any left over cruft, on a system crash.  The CONSTANT is badly named since we are just dropping an indicator file in this directory to tell CRI-O that the machine was rebooted, as opposed to somone just doing systemctl stop crio.  @mrunalp do you remember the exact reason?

Reply:
Yea I saw that path in the systemd units, I was too confused by the name

Reply:
Probably should be renamed DEFAULT_STATE_PATH or something like that.